lib/scarpe.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-if RUBY_VERSION[0..2] < "3.2"
-  $stderr.puts "Scarpe requires Ruby 3.2 or higher!"
-  exit(-1)
-end
+# if RUBY_VERSION[0..2] < "3.2"
```

Comment:
@noahgibbs If you're supportive of this, would it be better to delete than comment out?

Reply:
Huh. I think we should probably keep it. It's possible to run Scarpe without Bundler, and then I think we need the check.